CHANGELOG.md

```diff
@@ -9,6 +9,325 @@ Please see https://github.com/rapidsai/cudf/releases/tag/v0.20.0a for the latest
 # cuDF 0.19.0 (Date TBD)
 
 Please see https://github.com/rapidsai/cudf/releases/tag/v0.19.0a for the latest changes to this development branch.
+# cuDF 21.06.00 (Date TBD)
+
+Please see https://github.com/rapidsai/cudf/releases/tag/v21.06.00a for the latest changes to this development branch.
+
+# cuDF 0.19.0 (21 Apr 2021)
```

Comment:
can you check this file again? it doesn't look like the conflicts were resolved correctly. there should only be one `0.19.0` entry and the `0.20` entry should be replaced with `21.06.00`.

Reply:
Thanks @ajschmidt8 !